packages/cli/test/integration/generators/model.relation.integration.js

```diff
@@ -0,0 +1,77 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
```

Comment:
Please add this license header to all new files created in this pull request, with the exception of EJS templates.

Also the copyright year should be 2019 (not 2018).

Reply:
@bajtos, i probably need to run the `strong-tools` to update the license header for this repo. so we can do that in mid March.

Reply:
Done.